src/components/GlobalHeader.js

```diff
@@ -59,6 +59,11 @@ const GlobalHeader = ({ className }) => {
                 Community
               </ExternalLink>
             </li>
+            <li>
+              <Link href="/podcasts" className={styles.leftSideLink}>
```

Comment:
```suggestion
              <Link to="/podcasts" className={styles.leftSideLink}>
```

It looks like the `href` prop works, but it's still causing a page refresh. The `to` prop should make it work with the SPA.